packages/eslint-plugin-shopify/lib/config/rules/lodash.js

```diff
@@ -0,0 +1,84 @@
+// see https://github.com/wix/eslint-plugin-lodash
+
+module.exports = {
+  // Possible errors
+
+  // Use or avoid thisArg for Lodash method callbacks, depending on major version.
+  'callback-binding': 'warn',
+  // Use value returned from collection methods properly.
+  'collection-method-value': 'warn',
+  // Always return a value in iteratees of Lodash collection methods that aren't forEach.
+  'collection-return': 'error',
+  // Do not use .value() on chains that have already ended (e.g. with max() or reduce()) (fixable)
+  'no-double-unwrap': 'error',
+  // Do not use superfluous arguments on Lodash methods with a specified arity.
+  'no-extra-args': 'error',
+  // Prevent chaining without evaluation via value() or non-chainable methods like max().
+  'unwrap': 'error',
+
+  // Stylistic issues
+
+  // Enforce a specific chain style: explicit, implicit, or explicit only when necessary.
+  'chain-style': ['warn', 'explicit'],
```

Comment:
The reason I prefer explicit here is that 1) it's not obvious that the lodash object is itself a function that's usable, and if you import that object, you bring in all of lodash (whereas, we can eventually get to a point where we only import the methods from lodash we need and leave the rest of them out of the bundle, if we bring in only the functions themselves).